src/providers/MeetingProvider/docs/MeetingManager.stories.mdx

```diff
@@ -51,21 +28,38 @@ import { MeetingManagerConfig } from 'amazon-chime-sdk-component-library-react';
 (data: MeetingJoinData) => Promise<void>
 
 interface MeetingJoinData {
-  // The response of calling chime:CreateMeeting from your server.
-  meetingInfo: any;
-
-  // The response of calling chime:CreateAttendee from your server.
-  attendeeInfo: any;
-
-  // The kind of device for which the browser requests permission.
-  // Or override the default device label trigger in the Amazon Chime SDK for JavaScript.
-  deviceLabels?: DeviceLabels | DeviceLabelTrigger;
-
-  // Override the default event reporter in the Amazon Chime SDK for JavaScript.
-  eventReporter: EventReporter;
-
-  // Set the meeting config. This will override the props from MeetingProvider.
-  meetingManagerConfig: MeetingManagerConfig;
+  // Use the meeting session configuration.
+  // The default properties of the meeting session configuration can be updated to customize the meeting as per the builder's preference.
+  // Reference Meeting Session Configuration properties in Amazon Chime SDK for JavaScript: https://aws.github.io/amazon-chime-sdk-js/classes/meetingsessionconfiguration.html
+  meetingSessionConfiguration: MeetingSessionConfiguration;
+
+  options ?: {
+    // The kind of device for which the browser requests permission.
+    // Or override the default device label trigger in the Amazon Chime SDK for JavaScript.
+    deviceLabels?: DeviceLabels | DeviceLabelTrigger;
+
+    // Override the default event reporter in the Amazon Chime SDK for JavaScript.
+    eventReporter?: EventReporter;
+    
+    // The level of logging you want with your meeting session. 
+    // It is defined by the LogLevel (https://aws.github.io/amazon-chime-sdk-js/enums/loglevel.html) enum in `amazon-chime-sdk-js`.
+    logLevel?: LogLevel;
+    
+    // Configuration for setting up a MeetingSessionPOSTLogger (https://aws.github.io/amazon-chime-sdk-js/classes/meetingsessionpostlogger.html).
```

Comment:
Can you check whether these show as links correctly once storybook is loaded? If yes, please ignore this comment, if no, the syntax for linking would be:
`[MeetingSessionPOSTLogger](link)`

Have to also update other property links here if necessary.

Reply:
Unfortunately, they do not. Since, they are treated as JSX comments
![Screen Shot 2022-03-15 at 5 52 29 PM](https://user-images.githubusercontent.com/31940639/158495621-ea762149-43ae-446f-b905-b48ac1c9bed8.png)
.